sdk/identity/examples/client_certificate_credentials.rs

```diff
@@ -7,7 +7,7 @@ use azure_core::auth::TokenCredential;
 use azure_identity::{
     CertificateCredentialOptions, ClientCertificateCredential, DefaultAzureCredential,
 };
-use azure_security_keyvault::KeyClient;
+use azure_security_keyvault::KeyvaultClient;
```

Comment:
As with all our other language SDKs, we have separate clients for Keys and Cryptography, Secrets, and Certificates. They were named correctly already. These should idiomatically but consistently match our other SDKs unless there's a good reason not to.

Reply:
I see the confusion here.  I was trying to be consistent with other services thinking the KeyVault client was not consistent.

Consider Storage Queues, with [QueueServiceClient.get_queue_client](https://docs.microsoft.com/en-us/python/api/azure-storage-queue/azure.storage.queue.queueserviceclient?view=azure-python#azure-storage-queue-queueserviceclient-get-queue-client), or Service Bus, with [ServiceBusClient.get_queue_receiver](https://docs.microsoft.com/en-us/python/api/azure-storage-queue/azure.storage.queue.queueserviceclient?view=azure-python#azure-storage-queue-queueserviceclient-get-queue-client), or Storage Blobs with [BlobServiceClient.get_container_client](https://docs.microsoft.com/en-us/azure/developer/python/sdk/storage/azure-storage-blob/azure.storage.blob.blobserviceclient?view=storage-py-v12#get-container-client-container-)(...).[get_blob_client](https://docs.microsoft.com/en-us/azure/developer/python/sdk/storage/azure-storage-blob/azure.storage.blob.containerclient?view=storage-py-v12#get-blob-client-blob--snapshot-none-).